contrib/core-team/join-team.rst

```diff
@@ -5,12 +5,7 @@
    |purpose|
 
 
-[This is the existing core developers :ref:`become-core-developer` page from the devguide with the title changed.  We'll
+[This is the existing core developers :ref:`join-core-team` page from the devguide.  We'll
 adjust "core developer" to "core team" where appropriate.]
```

Comment:
Do we need to change this adjustment statement, since this PR is making a number of these adjustments?

Reply:
I think it can stay, because there are still other adjustments in the prose to make in followups, such as "and become a core developer of Python" -> "and join the Python core team".